src/Bundle/Cli/Dumper/DumperInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
```

Comment:
Should be changed and will be after merging https://github.com/Sylius/SyliusMailerBundle/pull/233 👌